src/UIComponents/ModelSummary.jsx

```diff
@@ -0,0 +1,51 @@
+/* React component to handle displaying a model summary. */
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { saveMessages, styles } from "../constants";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSpinner } from "@fortawesome/free-solid-svg-icons";
+
+class ModelSummary extends Component {
+  static propTypes = {
+    saveStatus: PropTypes.string
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    // Add a timer to simulate loading when saving a model.
+    let loadSpinner = () => this.setState({ isLoading: false });
+    setTimeout(loadSpinner, 1500);
```

Comment:
What happens if the save takes longer than 1.5 seconds?  Do we keep showing the spinner until it's actually done?

Reply:
I removed the timer functionality. It shows the spinner until the model is saved now.